src/main/java/core/basesyntax/service/ReportGeneratorImpl.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.service;
+
+import core.basesyntax.db.DataBase;
+
+import java.util.Map;
+
+public class ReportGeneratorImpl implements ReportGenerator {
```

Comment:
The class 'ReportGeneratorImpl' should be in the 'service.impl' package as per the checklist requirement to keep logic separated in corresponding packages.